src/mumble/MainWindow.cpp

```diff
@@ -166,7 +166,8 @@ MainWindow::MainWindow(QWidget *p) : QMainWindow(p) {
 	connect(qmUser, SIGNAL(aboutToShow()), this, SLOT(qmUser_aboutToShow()));
 	connect(qmChannel, SIGNAL(aboutToShow()), this, SLOT(qmChannel_aboutToShow()));
 	connect(qmListener, SIGNAL(aboutToShow()), this, SLOT(qmListener_aboutToShow()));
-	connect(qteChat, SIGNAL(entered(QString)), this, SLOT(sendChatbarMessage(QString)));
+	connect(qteChat, SIGNAL(entered(QString)), this, SLOT(sendChatbarText(QString)));
+	connect(qteChat, SIGNAL(pastedImage(QString)), this, SLOT(sendChatbarMessage(QString)));
```

Comment:
I think it would be better to insert the image into the current message, but without sending it. That way I can embed an image into my message.

And if a user wants to send only the image, they can just hit enter to do so :)

This would complicate the handling of Markdown though as we'd no longer be able to cleanly separate images from text while sending :thinking: